XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -47,12 +47,23 @@ public abstract class BaseControllerPointer : ControllerPoseSynchronizer,
         private Transform raycastOrigin = null;
 
         [SerializeField]
-        [Tooltip("The hold action that will enable the raise the input event for this pointer.")]
-        private MixedRealityInputAction activeHoldAction = MixedRealityInputAction.None;
+        [Tooltip("The hold actions that will enable the raise the input event for this pointer.")]
+        private MixedRealityInputAction[] holdActions = { MixedRealityInputAction.None };
+
+        [SerializeField]
+        [Range(0.01f, 1f)]
+        [Tooltip("The threshold before raising a single axis hold action.")]
+        private float holdThreshold = 0.25f;
```

Comment:
Maybe `holdPressThreshold`?

Reply:
I set it to 0.4 for my work framework

Reply:
Is this just a deadzone value?

Reply:
It's the value threshold that determines if the input down/up event is fired by the single axis action data. (I suppose it is a deadzone? 🤷‍♂️)